src/avl.py

```diff
@@ -0,0 +1,468 @@
+import json
+import logging
+
+from hypothesis import given
+import hypothesis.strategies as st
```

Comment:
adding this into the `if __name__ == "__main__"` [issue](https://github.com/tehgeekmeister/dadabass/issues/2).